epitome/dataset.py

```diff
@@ -276,16 +296,36 @@ def get_data_dir(data_dir=None):
         :return: directory containing data.h5 file
         :rtype: str
         '''
-
-        if not data_dir:
-            data_dir = GET_DATA_PATH()
-            download_and_unzip(S3_DATA_PATH, os.path.dirname(data_dir))
-
-        # make sure all required files exist
-        required_paths = [os.path.join(data_dir, x) for x in  REQUIRED_FILES]
-        assert(np.all([os.path.exists(x) for x in required_paths]))
-
-        return data_dir
+        # none are set --> default data path with default assembly
+        # both are set --> return data path with specified assembly
+        # only assembly is set --> return default data path with specified assembly
+        # only data path is set --> return just data path
+        default_data_dir = os.path.join(EpitomeDataset.get_epitome_user_path(), 'data')
+        epitome_data_dir = None
```

Comment:
why is this defined here if it is always defined below ?

Reply:
Removed